app/celery/nightly_tasks.py

```diff
@@ -136,14 +136,15 @@ def delete_notifications_for_service_and_type(service_id, notification_type, dat
         service_id,
         datetime_to_delete_before,
     )
-    end = datetime.utcnow()
-    current_app.logger.info(
-        f'delete-notifications-for-service-and-type: '
-        f'service: {service_id}, '
-        f'notification_type: {notification_type}, '
-        f'count deleted: {num_deleted}, '
-        f'duration: {(end - start).seconds} seconds'
-    )
+    if num_deleted:
```

Comment:
good point!

my only concern is that the celery stats won't really help as they don't tell much of the story (especially the split between letters and non letters), and i think re-adding isn't very helpful as if the tasks take super long as a one-off, re-adding the log line after the fact won't help if things did get deleted and then the next day's stuff might be a completely different shape. Especially considering it might have involved deleting test notifications. it's hard to tell if a service has historically sent test messages.

that said, i'm probs with you in that i'm not super wedded to these logs if we don't think they're giving us value. is it useful to know it took a long time to delete messages sent by a service that sent a lot of messages? probably not.

Reply:
i think i'll keep it, though you definitely made me think long and hard (perhaps too long 😏 )

👍